google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/UserDataConverter.java

```diff
@@ -118,6 +118,8 @@ static Value encodeValue(
       return Value.newBuilder().setIntegerValue((Long) sanitizedObject).build();
     } else if (sanitizedObject instanceof Double) {
       return Value.newBuilder().setDoubleValue((Double) sanitizedObject).build();
+    } else if (sanitizedObject instanceof Float) {
+      return Value.newBuilder().setDoubleValue(((Float) sanitizedObject)).build();
```

Comment:
Can we remove an extra bracket from here? like this: 
`return Value.newBuilder().setDoubleValue((Float) sanitizedObject).build();`

Reply:
Resolved.